src/ManualLink.php

```diff
@@ -237,74 +174,53 @@ private static function showForItem(CommonDBTM $item): void
             'ORDERBY'      => 'name'
         ]);
 
-        echo '<div class="spaced">';
-        echo '<table class="tab_cadrehov">';
-        echo '<tr>';
-        echo '<th colspan="2">';
-        echo self::getTypeName(Session::getPluralNumber());
-        echo '</th>';
-        echo '<th class="right">';
-       // Create a fake link to check rights.
-       // This is mandatory as CommonDBChild needs to know itemtype and items_id to compute rights.
-        $link = new self();
-        $link->fields['itemtype'] = $item->getType();
-        $link->fields['items_id'] = $item->fields[$item->getIndexName()];
-        if ($link->canCreateItem()) {
-            $form_url = self::getFormURL() . '?itemtype=' . $item->getType() . '&items_id=' . $item->fields[$item->getIndexName()];
-            echo '<a class="btn btn-primary" href="' . $form_url . '">';
-            echo '<i class="fas fa-plus"></i>&nbsp;';
-            echo _x('button', 'Add');
-            echo '</a>';
```

Comment:
This `Add` button is missing. This was the only link to manual link creation form.